### DISCUSSION TITLE
Error when configure Kiali on Istio with MTLS

### DISCUSSION THREAD

**Discussion by leandromoreirati** [kiali/kiali#7041] [Q&A]
### What do you want to discuss?
I need help to configure Kiali with Istio and cert-manager

### What does it improve / What problem does it solve?
Rencently I integrate Istio and cert-mange to manager certificates on Kubernetes, but when I deploy kiali I received the following erro mesange:

![image](https://github.com/kiali/kiali/assets/30784301/00c11bc7-5aac-4751-9f04-95c04d251abd)

![image](https://github.com/kiali/kiali/assets/30784301/40c653b6-4a5a-4241-9708-dea2891836e5)

Kiali not able to list Kubernetes resource (service, pods e etc)

![image](https://github.com/kiali/kiali/assets/30784301/aa6349be-679a-4672-a0aa-bdefb80453d7)

I already several re search on the web, but not found nothing could help me.

Environy### What do you want to discuss?
I need help to configure Kiali with Istio and cert-manager

### What does it improve / What problem does it solve?
Rencently I integrate Istio and cert-mange to manager certificates on Kubernetes, but when I deploy kiali I received the following erro mesange:

![image](https://github.com/kiali/kiali/assets/30784301/00c11bc7-5aac-4751-9f04-95c04d251abd)

![image](https://github.com/kiali/kiali/assets/30784301/40c653b6-4a5a-4241-9708-dea2891836e5)

Kiali not able to list Kubernetes resource (service, pods e etc)

![image](https://github.com/kiali/kiali/assets/30784301/aa6349be-679a-4672-a0aa-bdefb80453d7)

I already several re search on the web, but not found nothing could help me.

Can you help me please?

 Environment:
   Is Kubernetes:       true
   Kubernetes Version:  1.23.10
   Operator Version:    v1.78.0
   Istio control plane version: 1.14.4
   Istio data plane version: 1.14.4

Istio is installed and configured

![image](https://github.com/kiali/kiali/assets/30784301/e5995f51-96a9-47b1-a968-f15fb5e9769f)

Example of the application  application using ingress gateway:
![image](https://github.com/kiali/kiali/assets/30784301/dc3e4a34-51a6-4651-95a6-740e7338e3a4)

**Comment by nrfox** [comment]
@leandromoreirati it looks like your istio version is too old for the version of kiali you are trying to run. You can take a look at the compatibility matrix [here](https://kiali.io/docs/installation/installation-guide/prerequisites/#version-compatibility) and pick a version of kiali that is compatible with your version of istio. If you are still running into problems after getting the versions in sync then let us know.

**Comment by leandromoreirati** [comment]
@nrfox thanks for your response, really the istio vervion I used is very old.

I go upgrade istio version to 1.18 and adjust the kiali version and realize new tests and return when I finish my tests, one more time thanks.
